internal/pkg/service/stream/storage/node/coordinator/slicerotation/operator.go

```diff
@@ -186,7 +186,7 @@ func Start(d dependencies, config stagingConfig.OperatorConfig) error {
 }
 
 func (o *operator) checkSlices(ctx context.Context, wg *sync.WaitGroup) {
-	o.logger.Debugf(ctx, "checking slices upload conditions")
+	o.logger.Debugf(ctx, "checking slices rotation conditions")
 
```

Comment:
:x: 

There are "slice upload conditions", the slice state is swtiched from `writing -> closing` (then `-> uploading -> uploaded -> imported`)

Reply:
Fixed.